dom-renderables/DOMElement.js

```diff
@@ -204,7 +204,7 @@ DOMElement.prototype.onDismount = function onDismount() {
  * @return {undefined} undefined
  */
 DOMElement.prototype.onShow = function onShow() {
-    this.setProperty('display', 'block');
+    if (this._node.isShown()) this.setProperty('display', 'block');
```

Comment:
@alexanderGugel Thanks for fast responding!

Currently, once a parent node#show called, the parent node calls all the children's node#onShow and child nodes` DOMElement#onShow although some child nodes are manually hidden which should not be affected by a parent.

So, it can break child nodes' visible state, there's an example to see it
https://github.com/Famous/engine/issues/470

    1. Modify DOMElement.onShow not to change property when its Node.isShown() is false
    2. Modify Dispatch.show to ignore nodes which Node.isShown() is false
In that link, I chose the first way, but if I take the second to fix, it might be like this below.

Rather than changing DOMElement#onShow, changing Dispatch#show

Wrapping by ```if (node.isShown())``` statement
```javascript
if (node.isShown()) {
    if (node.onShow) node.onShow();

    var components = node.getComponents();
    for (var i = 0, len = components.length ; i < len ; i++)
        if (components[i] && components[i].onShow)
            components[i].onShow();
}
```

Both cases, updating Node#_shown value should be processed before than calling Dispatch#show

Thanks!